.github/workflows/test_sanity_check.yml

```
@@ -0,0 +1,86 @@
+name: TheRock Sanity Check
+
+on:
+  workflow_dispatch:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+  push:
+    branches:
+      - ADHOCBUILD
+
+jobs:
+  test_sanity_check:
+    name: "Sanity ROCM Test"
+    runs-on: linux-mi300-1gpu-ossci-rocm
+    strategy:
+      fail-fast: false
+    defaults:
+      run:
+        shell: bash
+    env:
+      VENV_DIR: ${{ github.workspace }}/.venv
+      ARTIFACT_RUN_ID: "${{ inputs.artifact_run_id != '' && inputs.artifact_run_id || github.run_id }}"
+      BUILD_ARTIFACTS_DIR: "build/artifacts"
+      THEROCK_BIN_DIR: "${{ github.workspace }}/build/artifacts/output_dir/bin"
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Install the AWS tool
+        uses: unfor19/install-aws-cli-action@27d6061dae5d39e89be4d2246824f15e111a7e06
```

```suggestion
        uses: unfor19/install-aws-cli-action@e8b481e524a99f37fbd39fdc1dcb3341ab091367 # v1.0.7
```
The commit you chose does not belong to a tagged release. The suggested on is the commit before the one you picked. The newest only touches a test workflow, so this isn't important for us.

Anyway, the action seems not to get too much attention. This might either indicate that it is quite stable _or_ that it is not actively maintained. Are there other alternatives?

---

I think an alternative is to use that script we wrote, which we can use. now that i think about it, i'll revert it back to the script we wrote since we know exactly what the code is doing and is stable

---

Switching back to the script SGTM.